pupil-auth/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:8.4.0
```

Comment:
IMO we should be pulling from node:8 - the latest is fine and will include security updates.